units/en/unit7/self-play.mdx

```diff
@@ -0,0 +1,137 @@
+
+Training against a set of slowly changing or unchanging adversaries with low diversity **results in more stable training. But a risk to overfit if the change is too slow.**
+
+We need then to control:
+
+- How **often do we change opponents** with `swap_steps` and `team_change` parameters.
+- The **number of opponents saved** with `window` parameter. A larger value of `window`
+ means that an agent's pool of opponents will contain a larger diversity of behaviors since it will contain policies from earlier in the training run.
+- **Probability of playing against the current self vs opponent** sampled in the pool with `play_against_latest_model_ratio`. A larger value of `play_against_latest_model_ratio`
+ indicates that an agent will be playing against the current opponent more often.
+- The **number of training steps before saving a new opponent** with `save_steps` parameters. A larger value of `save_steps`
+ will yield a set of opponents that cover a wider range of skill levels and possibly play styles since the policy receives more training.
+
+To get more details about these hyperparameters you definitely need [to check this part of the documentation](https://github.com/Unity-Technologies/ml-agents/blob/develop/docs/Training-Configuration-File.md#self-play)
+
+
+## The ELO Score to evaluate our agent
+
+### What is ELO Score?
+
+In adversarial games, tracking the **cumulative reward is not always a meaningful metric to track the learning progress:** because this metric is **dependent only on the skill of the opponent.**
+
+Instead, we’re using an ***ELO rating system*** (named after Arpad Elo) that calculates the **relative skill level** between 2 players from a given population in a zero-sum game.
+
+In a zero-sum game: one agent wins is the other agent loss. It’s a mathematical representation of a situation in which each participant’s gain or loss of utility **is exactly balanced by the gain or loss of the utility of the other participants.** We talk about zero-sum games because the sum of utility is equal to zero.
+
+During the training, this ELO (starting at a certain score: in general 1200), can decrease in the beginning but should then increase progressively.
+
+The Elo system is **inferred from the losses and draws against other players.** It means that player ratings depend **on the ratings of their opponents and the results scored against them.**
+
+Elo defines an Elo score that is the relative skills of a player in a zero-sum game. **We say relative because it depends on the performance of opponents.**
+
+The central idea is to think of the performance of a player **as a random variable that is normally distributed.**
+
+The difference in rating between 2 players serves as **the predictor of the outcomes of a match.** If the player wins but the probability is high it will not win a lot of points from their opponent, since it means that it was much stronger than it.
```

Comment:
```suggestion
The difference in rating between 2 players serves as **the predictor of the outcomes of a match.** If the player wins, but the probability of winning is high, it will only win a few points from its opponent since it means that it is much stronger than it.
```